learning_journal_basic/learning_journal_basic/tests.py

```diff
@@ -1,29 +1,219 @@
-import unittest
-
+import pytest
+import transaction
+import os
+import faker
+import random
 from pyramid import testing
+from .models import (
+    MyEntry,
+    get_engine,
+    get_session_factory,
+    get_tm_session,
+    )
+from .models.meta import Base
+from datetime import datetime
+from pyramid.httpexceptions import HTTPNotFound
+
+TITLE = ["learning journal 1", "learning journal 2", "learning journal 3"]
+
+FAKE = faker.Faker()
+
+ENTRIES = [MyEntry(
+    title=random.choice(TITLE),
+    blog_entry=FAKE.text(100),
+    creation_date=datetime.utcnow()
+    )
+        for i in range(3)]
 
+DB_SETTINGS = "sqlite:///:memory:" #setting DB (gloBAL VARIBLE)
 
-class ViewTests(unittest.TestCase):
-    def setUp(self):
-        self.config = testing.setUp()
+@pytest.fixture(scope="session")
+def setup_test_env():
+    os.environ[ #telling where the database_url test is
+        "DATABASE_URL"
+        ] = "sqlite:///:memory:"
```

Comment:
This can all be one line.  You are overwriting the value at of "DATABASE_URL" in the os.environ dict.